src/plugins/LADTable/LADTableCompositionPolicy.js

```diff
@@ -0,0 +1,39 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2020, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+export default class LADTableCompositionPolicy {
```

Comment:
Is it expected that this composition policy will have many instances? If not, perhaps change this to a function that returns an object rather than a class.

Reply:
Andrew had asked me to turn this one into a class, let's talk about it in standup and come to a consensus.

Reply:
An interesting point. I come from a Java background where everything is a class, so I automatically go to classes. 😄 

The two options proposed here are a Class or a factory function. Either will work fine in this situation. I think that ultimately they're doing the same thing - they're both defining a way to make instances of a type of object. JavaScript gives us a language construct for defining types of objects, ie. Classes, so my preference is to use that.

Using a Class here also gives us the option of extending it later if we wanted to. Policies have a well defined interface, so it also gives us the option of extending an abstract policy in future if we wanted to.